### PR TITLE
feat: recent stream activity.

### LINKED ISSUE #393: Recent Stream Activity
Show recent CFA and IDA events in a "Recent Activity" page/section in the app. Probably this can be put below the tokens table on the Wallet page

Screen Shot 2022-02-14 at 8 28 29 AM

### PR DESCRIPTION
**Pull request by zhyd1997**
Fixes #393

### CODE REVIEW

**Review by mikeghen — CHANGES_REQUESTED** [review]
I also think there's some missing icons from the frontend
Screen Shot 2022-03-03 at 10 06 25 AM

**Comment by mikeghen** [comment]
I left two comments, otherwise I think this is really good so thank you for taking a look at it @zhyd1997

File: package.json [hunk 1]
```diff
@@ -49,7 +50,8 @@
     "classnames": "2.2.6",
     "connected-react-router": "6.9.2",
     "cross-env": "7.0.3",
-    "ethers": "^5.5.4",
+    "ethers": "^5.5.2",
```
[the remark this answers is not in the document]
Comment (bigint) [reply package.json:53]: +1 if we downgrade this ENS name will not work 😕

[the remark this answers is not in the document]
Comment (zhyd1997) [reply package.json:53]: @mikeghen
not manually downgraded, maybe need update with v2 branch.

**Comment by zhyd1997** [comment]
> I also think there's some missing icons from the frontend Screen Shot 2022-03-03 at 10 06 25 AM

I have added all token address listed in `polygon-config.js`,  could you tell me which token you're using that missed in this PR?

**Comment by zhyd1997** [comment]
Updated: f37fb3b

Try again and see if the token displays.

**Review by mikeghen — APPROVED** [review]
Very good! great addition

**Comment by mikeghen** [comment]
@zhyd1997 there's one merge conflict, not sure what it is so have a look

**Comment by zhyd1997** [comment]
Conflicts resolved!

**Comment by zhyd1997** [comment]
I need to regenerate fonts, give me 10 minutes.

**Comment by Seroxdesign** [comment]
This is really good! Great work

**Comment by zhyd1997** [comment]
@mikeghen If you have any questions about the PR, please let me know

**Comment by zhyd1997** [comment]
Additional Checklist:

- [x] default token image
- [x] query unlisted token name -> use `hyperlink` instead.
- [x] filter out zeroAddress (0x0000...)
- [x] support copy address

🚫  show streaming market if possible

@mikeghen @elvijsTDL

**Comment by 0xSero** [comment]
This is really good!  Really nice to see how you used the superfluid sdk.

**Comment by zhyd1997** [comment]
Converting to draft PR again to consider if there a way to catch the stream that started from the Ricochet market.

**Comment by zhyd1997** [comment]
We can track started stream (or stop fixed stream) on our Launchpad according to the`flowsRecieved` and find it by this code:
https://github.com/Ricochet-Exchange/ricochet-frontend/blob/9a66f688f6eaef96a048e85cb59548594395e41d/src/store/main/selectors.ts#L7

just like the PR comments:
PR 1130 comments

But we will miss it if the stream is stopped, so I tried my best and it's hard to track the market by stream information.

**Comment by mikeghen** [comment]
This looks good @zhyd1997, I will merge it :+1:

**Pull request merged**